### PR TITLE
remove google analytics

### PR DESCRIPTION
**Pull request by virginiacc**
Remove Google Analytics code

## Removals

- rate checker GA tracking code
- references to GA tracking code files
- GA code in form explainer test fixture

## Review

- @cfpb/cfgov-frontends 
- @KimberlyMunoz 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by mthibos** [comment]
@virginiacc  Why are we removing google analytics code on OaH?

**Comment by virginiacc** [comment]
@mthibos Don't worry, it's not going away altogether! At the moment we're using both Google Analytics and Google Tag Manager, and the plan is to consolidate all the tracking in GTM. So any code that references GA is being removed and will be replaced in GTM.

**Comment by mthibos** [comment]
@virginiacc   So will the new GTM code be deployed at the same time as this PR?  I'd hate to have a gap in our analytics.

**Comment by KimberlyMunoz** [comment]
LGTM :+1:

File: test/js/fixtures/form-explainer.html [hunk 1]
```diff
@@ -41,30 +41,12 @@
...
-        (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
-        g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
-        s.parentNode.insertBefore(g,s)}(document,'script'));
-    </script>
+    
 
 </head>
 
 <body>
-    <!-- Google Tag Manager -->
```
Comment (sebworks) [inline test/js/fixtures/form-explainer.html]: Did you intend to remove the Tag Manger code?

Comment (virginiacc) [inline test/js/fixtures/form-explainer.html]: Since it's a fixture, it shouldn't need the tracking code.

**Comment by KimberlyMunoz** [comment]
Hey @virginiacc and @contolini, I was going to prepare a release today. Does this PR need to go out with the other remove-analytics PRs?

**Pull request merged**